about.md

```diff
@@ -31,5 +31,11 @@ We also have a [Slack channel][2] on the University of York Slack workspace, whe
 any time to get help and advice. This is currently only available to staff and research
 postgraduate students.
 
+Finally, we have a [Google Calendar][3] that contains information about our events. You should get access to
+this when joining the mailing list!
```

Comment:
```suggestion
Finally, we have a [Google Calendar][3] that contains information about our events:
```

Reply:
Pushed resolving commit before realising that I can commit the suggest via the web interface!